spec/services/remove_section_service_spec.rb

```diff
@@ -70,9 +74,14 @@
       expect(manual).not_to have_received(:draft)
     end
 
-    it "does not notifies its listeners" do
+    it "does not export a manual" do
+      begin; service.call; rescue; end
```

Comment:
Do you know why we need this `rescue` block? It's a bit horrible! However, it looks as if you've just copied it from an existing spec. If so, it would be nice to explain that in the commit note and perhaps to create a new issue to investigate removing these `rescue` blocks.